data/data/vsphere/master/main.tf

```diff
@@ -1,31 +1,47 @@
 locals {
+  // limitation of baremetal-runtimecfg.  The hostname must be master
```

Comment:
Is this correct? `baremetal-runtimecfg`

Reply:
I tried to find more information on this, it was there since the creation of the file.
@jcpowermac do you know more?

Reply:
@e-tienne I can't find a ref to "master" anymore within:
https://github.com/openshift/baremetal-runtimecfg
Let's remove it and see if CI passes.